ci-operator/step-registry/openshift/image-based/install/openshift-image-based-install-commands.sh

```diff
@@ -40,8 +40,8 @@ set -euo pipefail
 
 export SEED_IMAGE="${SEED_IMAGE}"
 export SEED_VERSION="${SEED_VERSION}"
-export LCA_IMAGE="${LCA_PULL_REF}"
```

Comment:
This PR looks good to me, but I'm not sure I have the full context on some of these scripts, was this not being used at all or were any of the lca scripts using this env var?

Reply:
The [ib-cli](https://github.com/openshift-kni/lifecycle-agent/blob/main/ib-cli/README.md) is being deprecated in favor of the `openshift-install image-based` commands in OCP 4.17. Before, we used this env var to [extract](https://github.com/rh-ecosystem-edge/ib-orchestrate-vm/blob/671670f83cc761eed213c372ead8b480bbd81529/Makefile.ibi#L57) the `ib-cli` from and it was also an [argument](https://github.com/rh-ecosystem-edge/ib-orchestrate-vm/blob/671670f83cc761eed213c372ead8b480bbd81529/Makefile.ibi#L74) to the `ib-cli create-iso` command. Since we don't need the` ib-cli`, but only the `openshift-install`, this env var becomes redundant. Does this make sense?

Reply:
I see, yup, thats what I was curious about, thanks for the context @mresvanis !